pkg/file/transactionset.go

```diff
@@ -94,7 +94,7 @@ func (r *TransactionSet) Validate(transRule *rules.TransactionRule) error {
 
 				if err = r.Loops[segIndex].Validate(&rule); err != nil {
 					if rules.IsMaskRequired(rule.Mask) {
-						log.Println(err)
+						util.Log().Debug().Log("Validation Error: " + err.Error())
 						return fmt.Errorf("loop(%02d) should have valid %s loop, %s", segIndex, strings.ToLower(rule.Name), err.Error())
```

Comment:
The library should return errors wrapped with an appropriate description. Let's remove the logging from `pkg/...`